Makefile

```diff
@@ -51,8 +51,9 @@ clean:
 .PHONY: html
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+        find $(BUILDDIR)/html -name "*.drawio" -print | xargs -I {} draw.io -x -f "png" {}
 	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html & drawio images are in source/images."
```

Comment:
Revert this message to just: `"Build finished. The HTML pages are in $(BUILDDIR)/html."`